README.md

```diff
@@ -1 +1,145 @@
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.hostinger.com/tutorials/what-is-html">HTML</a></li>
+    <li><a href="https://www.hostinger.com/tutorials/what-is-css">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Follow a correct Gitflow**
+- **Comply with linters**
+- **Show a greeting message**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://carlosigreda.github.io/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+### Prerequisites
+
+In order to run this project you need:
+
+✅ Github account <br>
+✅ Visual Studio Code installed <br>
+✅ Node.js installed <br>
+✅ Git Bash installed (optional)   
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd [my-folder]
+  git clone git@github.com:CarlosIgreda/HelloMicroverseProject.git
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="author"></a>
+
+- GitHub: [@CarlosIgreda](https://github.com/CarlosIgreda)
+- Twitter: [@carlosigreda](https://twitter.com/carlosigreda)
+- LinkedIn: [carlos-jesús-igreda-rivas-323763116](https://www.linkedin.com/in/carlos-jes%C3%BAs-igreda-rivas-323763116/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Interaction using Javascript**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can follow me on Github.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all Microverse staff and my learning partners as well.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ <a name="faq"></a>
+
+- **What is a Linter?**
+
+  - Linter is a tool that analyzes source code to flag programming errors, bugs, stylistic errors, and suspicious constructs.
+
+- **What is meant by Gitflow?**
+
+  - Gitflow is an alternative Git branching model that involves the use of feature branches and multiple primary branches.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [Optional] Dear @CarlosIgreda, Notice the link for your MIT file is broken, Please make sure to add the MIT license file and update its content in regard to your name and project. Also replace the `[YOUR NAME]` part with your name, and `[APP TYPE]` with your app type.